src/images/library/code/broad-square.svg

```diff
@@ -0,0 +1,10 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 44.3 47">
+    <defs>
+        <path id="a" d="M1.9 2.1H42v43.4H1.9V2.1z"/>
```

Comment:
These ids should be given more unique names, to make sure they don't overlap with other svgs.